src/app/Http/Controllers/SupportPageController.php

```diff
@@ -39,15 +39,18 @@ public function show(): View
             );
         }
 
+        $path = preg_replace("/[^0-9.]/", '', base_path());
+
         return view('support-page::show')
             ->with('list', [
                 'Name' => config('app.name'),
                 'Acronym' => config('app.acronym'),
-                'Build' => config('app.build'), // TODO Automatic read from parent folder
+                'Build' => $path,
                 'Laravel' => app()->version(),
                 'PHP' => phpversion(),
             ])
-            ->with('groups', $groups);
+            ->with('groups', $groups)
+            ->with('title', 'Support');
```

Comment:
Why not make the page title configurable?

Reply:
I have removed title from the controller and put it back on the blade

Reply:
No, I meant make it a config option.